app/src/main/java/org/mozilla/focus/fragment/DownloadsFragment.java

```diff
@@ -30,18 +34,31 @@ public static DownloadsFragment newInstance() {
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         recyclerView = (RecyclerView) inflater.inflate(R.layout.fragment_downloads, container, false);
+        mDownloadListAdapter = new DownloadListAdapter(recyclerView.getContext());
```

Comment:
In `onCreate`, the BroadcastReceiver is created after creation of DownloadListAdapter, how come the adapter becomes null?